config.js

```diff
@@ -17,10 +19,21 @@ exports.client = {
 }
 
 exports.github = {
+  'dev': {
+    'teamId': '1940407', // github.com/dvcoders-test
+    'apiKey': process.env.GITHUB_API_KEY_DEV || null
```

Comment:
This other variable is optional. There's a conditional here which allows the developer to toggle what environment they're working in (https://github.com/dvcoders/dvcoders-backend/pull/38/files#diff-0364f57fbff2fabbe941ed20c328ef1aR39).

So if you don't want to set the dev API keys and tokens, you can choose not to.